### DISCUSSION TITLE
How to Visualize TitanGraph with Berkely DB as back end(.jdb file extension)

### DISCUSSION THREAD

**Discussion by Rahamathulla** [gephi/gephi#2371] [Q&A] [answered]
## Expected Behavior
is there any way to visualize the 
Titan Graph with berkely db as back end.
## Current Behavior
NA
## Possible Solution
Create Plugin

## Steps to Reproduce

1.Open Gephi
2.Try to open the database file(with extn .jdb)
3.not be able to open the database
4.

## Context
Unable to view the graph database

## Your Environment

* Version used: Gephi 0.9.2
* Java version: 1.8
* Operating System: windows 10

**Comment by eduramiba** [comment, marked as answer]
You will need to export your graph to a compatible format. Maybe you can use gremlin/tinkerpop GraphMLWriter.

**Comment by Rahamathulla** [comment]
Can you please explain the steps for exporting to Gephi compatible format. Thanks in advance
